.github/workflows/ci.yml

```diff
@@ -161,10 +161,10 @@ jobs:
             args: --all-targets
             download_sway_artifacts: sway-examples
           - cargo_command: clippy
-            args: --all-targets --all-features
+            args: --all-targets --features "default fuel-core-lib test-type-paths"
             download_sway_artifacts: sway-examples-w-type-paths
           - cargo_command: nextest
-            args: run --all-targets --all-features --workspace
+            args: run --all-targets --features "default fuel-core-lib test-type-paths" --workspace
```

Comment:
I don't see why we should stop testing all the other features.

Reply:
https://github.com/FuelLabs/fuels-rs/pull/998#discussion_r1226057805

Reply:
I think that all other features are being tested because each of our packages has a `default` and exports all its features through it. 🤔